src/main/kotlin/org/rust/lang/core/resolve2/ModCollectorBase.kt

```diff
@@ -50,6 +50,8 @@ class ModCollectorBase private constructor(
         for (item in items) {
             if (item !is RsExternCrateItemStub) {
                 macroIndexInParent += collectElement(item, macroIndexInParent)
+            } else {
+                macroIndexInParent++
             }
```

Comment:
Maybe simplify this since `hasMacroIndex` is fast now

```suggestion
            if (item !is RsExternCrateItemStub) {
                collectElement(item, macroIndexInParent)
            }
            if (item.hasMacroIndex()) {
                macroIndexInParent++
            }
```